fixtures/ipns_records/README.md

```diff
@@ -4,20 +4,13 @@
 
 ### Fixtures and ipns-record
 
-```sh
-# using Kubo CLI version 0.21.0-rc3 (https://dist.ipfs.tech/kubo/v0.21.0-rc3/)
-FILE_CID=$(echo "Hello IPFS" | ipfs add --cid-version 1 -q)
-IPNS_KEY=$(ipfs key gen ipns-record)
+```shell
+> go run generator/main.go
 
-ipfs dag export ${FILE_CID} > fixtures.car
-
-# publish a record valid for a 100 years
-ipfs name publish --key=ipns-record --quieter --ttl=876600h --lifetime=876600h /ipfs/${FILE_CID}
-ipfs routing get /ipns/${IPNS_KEY} > ${IPNS_KEY}.ipns-record
-
-echo IPNS_KEY=${IPNS_KEY}
-echo FILE_CID=${FILE_CID} # A file containing "Hello IPFS"
-
-# IPNS_KEY=k51qzi5uqu5dh71qgwangrt6r0nd4094i88nsady6qgd1dhjcyfsaqmpp143ab
-# FILE_CID=bafkreidfdrlkeq4m4xnxuyx6iae76fdm4wgl5d4xzsb77ixhyqwumhz244 # A file containing Hello IPFS
-```
+k51qzi5uqu5dm4tm0wt8srkg9h9suud4wuiwjimndrkydqm81cqtlb5ak6p7ku_v1.ipns-record -> /ipfs/bafkqadtwgeww63tmpeqhezldn5zgi
+k51qzi5uqu5dlkw8pxuw9qmqayfdeh4kfebhmreauqdc6a7c3y7d5i9fi8mk9w_v1-v2.ipns-record -> /ipfs/bafkqaddwgevxmmraojswg33smq
+k51qzi5uqu5dlmit2tuwdvnx4sbnyqgmvbxftl0eo3f33wwtb9gr7yozae9kpw_v1-v2-broken-v1-value.ipns-record -> /ipfs/bafkqahtwgevxmmraojswg33smqqho2lunaqge4tpnnsw4idwmfwhkzi
+k51qzi5uqu5dilgf7gorsh9vcqqq4myo6jd4zmqkuy9pxyxi5fua3uf7axph4y_v1-v2-broken-signature-v1.ipns-record -> /ipfs/bafkqahtwgevxmmrao5uxi2bamjzg623fnyqhg2lhnzqxi5lsmuqhmmi
+k51qzi5uqu5diamp7qnnvs1p1gzmku3eijkeijs3418j23j077zrkok63xdm8c_v1-v2-broken-signature-v2.ipns-record -> /ipfs/bafkqahtwgevxmmrao5uxi2bamjzg623fnyqhg2lhnzqxi5lsmuqhmmq
+k51qzi5uqu5dit2ku9mutlfgwyz8u730on38kd10m97m36bjt66my99hb6103f_v2.ipns-record -> /ipfs/bafkqadtwgiww63tmpeqhezldn5zgi
+```
```

Comment:
If we have to fix & run this code one 2 years from now for example, is there a way to tell which parts of the codebase to update?